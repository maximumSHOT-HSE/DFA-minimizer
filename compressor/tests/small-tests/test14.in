5 10 2
1 3 0
1 2 1
2 1 0
2 3 1
3 4 0
3 4 1
4 4 0
4 3 1
5 1 0
5 3 1
5
2
1 4 
