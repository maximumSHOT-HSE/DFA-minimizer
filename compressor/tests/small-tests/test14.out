4 8 2
2 1 0
1 2 1
1 3 0
2 3 1
4 3 1
3 4 0
4 4 0
3 4 1
2
2
1 4 
